config/environments/development.rb

```diff
@@ -37,6 +37,9 @@
     # Expands the lines which load the assets
     config.assets.debug = true
   end
+
+  # Raise exception on mass assignment protection for Active Record models
```

Comment:
http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-3-1-to-rails-3-2-config-environments-test-rb